main/accounts/types.ts

```diff
@@ -72,6 +73,16 @@ export interface Approval {
   approve: (data: any) => void
 }
 
+export interface Permit {
+  deadline: string | number
+  spender: string
+  value: string | number
+  owner: string
+  verifyingContract: string
+  chainId: string | number
```

Comment:
let's make sure for all of these types we know all the formats they can take. eg. can the chain id be a hex string?